### PR TITLE
Clean-up: go generate

### PR DESCRIPTION
**Pull request by toaster**
### Description:

As part of a general code clean-up during OpenGL experiments, I ran `go generate ./...` top level.
I fixed some issues with this and used the latest stringer.
Some mobile generators failed for me because I don’t have an Android SDK environment.
For the same reason, I am not able to test the changes to the mobile `binres_string.go` and `code_string.go` files. The changes look good to me. Any help with testing them is appreciated.

### Checklist:

- [ ] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: internal/driver/mobile/event/key/code_string.go [hunk 1]
```diff
@@ -1,60 +1,183 @@
...
+	_Code_name_2 = "CodeSemicolonCodeApostropheCodeGraveAccentCodeCommaCodeFullStopCodeSlashCodeCapsLockCodeF1CodeF2CodeF3CodeF4CodeF5CodeF6CodeF7CodeF8CodeF9CodeF10CodeF11CodeF12"
+	_Code_name_3 = "CodePauseCodeInsertCodeHomeCodePageUpCodeDeleteForwardCodeEndCodePageDownCodeRightArrowCodeLeftArrowCodeDownArrowCodeUpArrowCodeKeypadNumLockCodeKeypadSlashCodeKeypadAsteriskCodeKeypadHyphenMinusCodeKeypadPlusSignCodeKeypadEnterCodeKeypad1CodeKeypad2CodeKeypad3CodeKeypad4CodeKeypad5CodeKeypad6CodeKeypad7CodeKeypad8CodeKeypad9CodeKeypad0CodeKeypadFullStop"
+	_Code_name_4 = "CodeKeypadEqualSignCodeF13CodeF14CodeF15CodeF16CodeF17CodeF18CodeF19CodeF20CodeF21CodeF22CodeF23CodeF24"
+	_Code_name_5 = "CodeHelp"
+	_Code_name_6 = "CodeMuteCodeVolumeUpCodeVolumeDown"
+	_Code_name_7 = "CodeLeftControlCodeLeftShiftCodeLeftAltCodeLeftGUICodeRightControlCodeRightShiftCodeRightAltCodeRightGUI"
+	_Code_name_8 = "CodeCompose"
```
Comment (changkun) [inline internal/driver/mobile/event/key/code_string.go:142]: FYI: While I merge gomobile in, these names were rewritten to meet the CI lint requirements. Currently, it seems the requirement was removed, and these naming style is acceptable :)

Comment (changkun) [reply internal/driver/mobile/event/key/code_string.go:142]: Same in other places.

**Review by Bluebugs — APPROVED** [review]

**Comment by andydotxyz** [comment]
> Some mobile generators failed for me because I don’t have an Android SDK environment.
> For the same reason, I am not able to test the changes to the mobile `binres_string.go` and `code_string.go` files. The changes look good to me. Any help with testing them is appreciated.

I have tried but gomobile generation does not succeed - there is a "go generate" using `gomobile` to create docs that we don't use and have replaced. If you can remove the "go generate" from cmd/fyne/internal/mobile/main.go then it will work OK.

**Review by andydotxyz — CHANGES_REQUESTED** [review]
As noted in the other comment, "go generate gomobile..." in cmd/fyne/internal/mobile/main.go is blocking this from working as expected on mobile. That should be the last change.

**Review by andydotxyz — APPROVED** [review]
Thanks so much for this

**Pull request merged**